src/XamlStyler.Extension.Windows.VS2019/XamlStyler.Package.vsct

```diff
@@ -35,12 +35,6 @@
       <Group guid="GuidXamlStylerMenuSet" id="GroupIDXamlStylerMenu" priority="0x0001">
         <Parent guid="GuidVisualStudioXAMLWindowContextMenu" id="IDVisualStudioXAMLWindowContextMenu"/>
       </Group>
-      <Group guid="GuidXamlStylerMenuSet" id="GroupIDXamlStylerMenu" priority="0x0001">
-        <Parent guid="GuidVisualStudioXamarWindowContextMenu" id="IDVisualStudioXamarinWindowContextMenu"/>
```

Comment:
@NicoVermeir do you recall why we added these Xamarin-specific entries? Are they still relevant?

Reply:
Looks like they were there for AXAML support :/